.github/workflows/validate_pr_labels_and_release_notes.yml

```diff
@@ -56,15 +85,85 @@ jobs:
               cmd="awk '/<$LABEL>.*<\/$LABEL>/'"
               FEATURE_RELEASE_NOTE="$(echo $PR_BODY | eval $cmd)"
               if [[ -z "$FEATURE_RELEASE_NOTE" ]]; then
-                 echo -e "${RED}Pull requests labeled with '$LABEL' must have the release note in the pull request body in the following format '<$LABEL> {THE RELEASE NOTE}  </$LABEL>'${NC}"
-                 SHOULD_FAIL_JOB="true"
+                 ERROR_MESSAGE="❌ Pull requests labeled with '${LABEL}' must have the release note wrapped in the same xml tag."
```

Comment:
Shall we keep the info about which label to use?
It might help unfamiliar contributors

Reply:
I totally failed to post a comment with properly escaped backticks and xml tags so I just gave up. I don't mind having extra help here :)